packages/frontend-v2/components/Error/ErrorBoundary.tsx

```diff
@@ -1,21 +1,22 @@
-import { ErrorBoundary as ReactErrorBoundary } from "react-error-boundary";
-import { logger } from "../../utils";
-import { ClientSideError } from "./ClientSideError";
+// import { ErrorBoundary as ReactErrorBoundary } from "react-error-boundary";
+import React, { PropsWithChildren } from "react";
+// import { logger } from "../../utils";
+// import { ClientSideError } from "./ClientSideError";
 
-const clientSideErrorHandler = (
-  error: Error,
-  { componentStack }: { componentStack: string }
-) => {
-  logger.error(error.message, componentStack);
-};
+// const clientSideErrorHandler = (
+//   error: Error,
+//   { componentStack }: { componentStack: string }
+// ) => {
+//   logger.error(error.message, componentStack);
+// };
 
-export const ErrorBoundary: React.FC = ({ children }) => {
+export const ErrorBoundary: React.FC<PropsWithChildren> = ({ children }) => {
   return (
-    <ReactErrorBoundary
-      FallbackComponent={ClientSideError}
-      onError={clientSideErrorHandler}
-    >
-      {children}
-    </ReactErrorBoundary>
+    // <ReactErrorBoundary
+    //   FallbackComponent={ClientSideError}
+    //   onError={clientSideErrorHandler}
+    // >
+    <div>{children}</div>
+    // </ReactErrorBoundary>
```

Comment:
error boundary library is incompatible with react 18, will switch out for a compatible one and style it in a separate PR